examples/customer-behavior/main.bal

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/oauth2;
+import ballerinax/hubspot.crm.properties as hsproperties;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+
+hsproperties:OAuth2RefreshTokenGrantConfig auth = {
+    clientId: clientId,
+    clientSecret: clientSecret,
+    refreshToken: refreshToken,
+    credentialBearer: oauth2:POST_BODY_BEARER
+};
+
+final string groupName = "customer_behavior";
+final string customPropertyName = "purchase_frequency_property";
+final string dependantPropertyName = "preferred_channel_property";
+final hsproperties:Client hubSpotProperties = check new ({auth});
```

Comment:
Let's reorder the code like this as this is more intuitive. Please check other places as well.
```suggestion
};
final hsproperties:Client hubSpotProperties = check new ({ auth });

final string groupName = "customer_behavior";
final string customPropertyName = "purchase_frequency_property";
final string dependantPropertyName = "preferred_channel_property";
```